stdlib/LinearAlgebra/src/givens.jl

```diff
@@ -339,17 +342,16 @@ function lmul!(G::Givens, A::AbstractVecOrMat)
     end
     return A
 end
-function rmul!(A::AbstractMatrix, adjG::Adjoint{<:Any,<:Givens})
+function rmul!(A::AbstractMatrix, G::Givens)
     @assert !has_offset_axes(A)
-    G = adjG.parent
     m, n = size(A, 1), size(A, 2)
     if G.i2 > n
         throw(DimensionMismatch("column indices for rotation are outside the matrix"))
     end
     @inbounds @simd for i = 1:m
         a1, a2 = A[i,G.i1], A[i,G.i2]
-        A[i,G.i1] =  a1*G.c + a2*conj(G.s)
```

Comment:
Strange that we used to take the `conj` of `G.s` here but not in the next line.  Was this an observable bug?

Reply:
It's how the Given's rotation is defined, so should be correct